web/src/views/SubmissionPortal/HarmonizerView.vue

```diff
@@ -386,7 +387,26 @@ export default defineComponent({
     }
 
     async function validate() {
-      const data = harmonizerApi.exportJson();
+      const data = harmonizerApi.exportJson(); // Gets data from harmonizer API
+
+      // Check if the spreadsheet is empty
+      const isEmpty = Object.keys(data).length === 0;
+      // Update invalid cells if empty
+      if (isEmpty) {
+        invalidCells.value = {
+          ...invalidCells.value,
+          [activeTemplateKey.value]: data,
+        };
+        tabsValidated.value = {
+          ...tabsValidated.value,
+          [activeTemplateKey.value]: false,
+        };
+        snackbar.value = Object.values(tabsValidated.value).every((value) => value);
```

Comment:
There's no need to compute this here. Given the way `tabsValidated` is set in the line above, this will always result in `snackbar.value = false`. 

Since we now have several possible snackbar displays, it also might be more clear if we rename `snackbar` to something like `validationSuccessSnackbar`. Or, extra credit, is it possible to store a `ref` that contains the full snackbar state (message, color, timeout) and then only render one `v-snackbar` component?